src/WebUI/Workloads/OtherWorkloadsTable.tsx

```diff
@@ -124,14 +123,14 @@ export class OtherWorkloads extends React.Component<IOtherWorkloadsProperties, I
         const storeState = this._store.getState();
         this.setState({
             statefulSets: storeState.statefulSetList && storeState.statefulSetList.items || []
-        });
+        }, this._onMetadataFetch);
```

Comment:
one idea i have in this case is, have a dictionary with 4 values, like a["daemon"]=false etc, then at each of the call set the value to true

in the final _onMetadataFetch, check for all keys of the dictionary and see value is true and then call the original one, so you do not have to rely on === 4

Reply:
Done.